src/services/schedule.ts

```diff
@@ -108,13 +108,43 @@ export async function startSchedule(
         await waitForTasksToFinish(server!)
     }
 
-    return stopSchedule
+    const reloadSchedule = async () => {
+        lockTimeout && clearTimeout(lockTimeout)
+        weHaveTheLock = false
+        await lock?.unlock().catch(Sentry.captureException)
+
+        server.pluginSchedule = await loadPluginSchedule(piscina)
+
+        lockTimeout = setTimeout(tryToGetTheLock, 0)
```

Comment:
Is this redlock unlocking and locking intentional? This will pause all scheduled tasks and potentially cause some to be missed. Very likely it will also be a different plugin server instance that picks up the lock and resumes the work.

Reply:
So there's a problem in general where having the plugin server be down during HOUR:00 will make it so no queued tasks get run.

I think there needs to be a long-term solution for that but until then I think we can change this to just await until the plugins have (re)loaded